### ISSUE TITLE
Can't set "appendTo" property to null

### ISSUE THREAD

**Issue by hujhax** [zensh/ui-autocomplete#23]
From [the documentation for appendTo](http://api.jqueryui.com/autocomplete/#option-appendTo):

> When the value is `null`, the parents of the input field will be checked for a class of `ui-front`. If an element with the `ui-front` class is found, the menu will be appended to that element. Regardless of the value, if no element is found, the menu will be appended to the body.

It looks like this line of code is making this impossible:

    options.appendTo = options.appendTo || element.parents('.ng-view')[0] || element.parents('[ng-view]')[0] || null;

**Comment by zensh** [comment]
If you want to append to body, you can set it directly

**Comment by hujhax** [comment]
True, but what I want is to append to the nearest parent that ends in `ui-front`.
